code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -20,6 +20,7 @@ var/global/list/image/splatter_cache=list()
 	var/basecolor="#A10808" // Color when wet.
 	var/list/datum/disease2/disease/virus2 = list()
 	var/amount = 5
+	appearance_flags = NO_CLIENT_COLOR
```

Comment:
i normally wouldn't advise things on the premise of "eh just fix it later" but in this case it's a single line which has no real alternative